### PR TITLE
dev-python/pybind11: fix cmake config files

### PR DESCRIPTION
**Pull request by waebbl**
Cmake config files were not present in 2.4.3, but are needed by some
dependant ebuilds. The patch adds support for installing the cmake
config files.

Closes: https://bugs.gentoo.org/706070
Package-Manager: Portage-2.3.86, Repoman-2.3.20
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

Necessary profile changes to mask doc USE flag on powerpc/ppc32 and arm64 due to dev-python/breathe not keyworded on those arches.

### CODE REVIEW

**Comment by waebbl** [comment]
update to resolve conflicts which have been evolved

File: profiles/arch/arm64/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Bernd Waibel <waebbl@gmail.com> (2020-03-31)
+# dev-python/breathe is not keyworded
```
Comment (thesamesam) [inline profiles/arch/arm64/package.use.mask:5]: Fancy feeling a keyword request even if you do mask it?

File: profiles/arch/powerpc/ppc32/package.use.mask [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Bernd Waibel <waebbl@gmail.com> (2020-03-31)
+# dev-python/breathe is not keyworded
```
Comment (thesamesam) [inline profiles/arch/powerpc/ppc32/package.use.mask:5]: It is now.

**Comment by waebbl** [comment]
Resolve conflict. Also don't drop old version, as it's in a stabilization process and already has amd64 and x86 stable.

**Comment by juippis** [comment]
079a605885d08aa1bfaa8a952eb0491a0253f106

**Pull request closed without merging**